libs/cart/src/actions/cart.actions.ts

```diff
@@ -95,6 +97,18 @@ export class DaffResolveCart implements Action {
 	readonly type = DaffCartActionTypes.ResolveCartAction;
 }
 
+export class DaffResolveCartSuccess<T extends DaffCart = DaffCart> implements Action {
+  readonly type = DaffCartActionTypes.ResolveCartSuccessAction;
+
+  constructor(public payload: Partial<T>) {}
```

Comment:
I don't think this should have a payload. The Actions that trigger the ResolveCartSuccess should handle setting the cart on state. This action should just set the resolve boolean

Reply:
changed